ui/analyse/src/view/components.ts

```diff
@@ -385,15 +386,15 @@ export const renderMaterialDiffs = (ctrl: AnalyseCtrl): [VNode, VNode] =>
 export const addChapterId = (study: StudyCtrl | undefined, cssClass: string) =>
   cssClass + (study && study.data.chapter ? '.' + study.data.chapter.id : '');
 
-export function makeChat(ctrl: AnalyseCtrl, insert: (chat: HTMLElement) => void) {
+export function makeChatEl(ctrl: AnalyseCtrl, insert: (chat: HTMLElement) => void) {
   if (ctrl.opts.chat) {
     const chatEl = document.createElement('section');
     chatEl.classList.add('mchat');
     insert(chatEl);
     const chatOpts = ctrl.opts.chat;
-    chatOpts.instance?.then(c => c.destroy());
+    chatOpts.instance?.destroy();
     chatOpts.enhance = { plies: true, boards: !!ctrl.study?.relay };
-    chatOpts.instance = site.makeChat(chatOpts);
+    chatOpts.instance = makeChat(chatOpts);
```

Comment:
we lost lazy-loading of the chat as well. I suppose that's ok as pages with a chat usually load it right away.

Reply:
chat is only 12k uncompressed. i'm not sure the amount of browser memory we're saving for people who aren't on a chat ui is worth the async overhead.

Reply:
The point was to load the essential game UI as fast as possible. Each game is played in a new page and getting a functional chess board as fast as possible on low powered devices was a goal.